packager/media/base/raw_key_source.cc

```diff
@@ -56,9 +57,9 @@ Status RawKeySource::GetKey(const std::vector<uint8_t>& key_id,
       return Status::OK;
     }
   }
-  std::string key_id_str(key_id.begin(), key_id.end());
   return Status(error::INTERNAL_ERROR,
-                "Key for key_id=" + absl::BytesToHexString(key_id_str) +
+                "Key for key_id=" +
+                    absl::BytesToHexString(byte_vector_to_string_view(key_id)) +
```

Comment:
One pattern that repeated many times was changing `base::HexEncode(data*,size) ->  absl::BytesToHexString(string_view)`

The refactorings so far were done like here, but the problem is that it performs an extra copy+allocation.
What worked for me was constructing a string_view directly from the uint8_t array/vector. Sadly, with string_view the constructors can't take in a uint8_t* so a `reinterpret_cast` is needed. Created helper functions for it.